app/services/participants/retry_continue_training.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+module Participants
+  class RetryContinueTraining < BaseService
+    def call
+      clear_participant_continue_training_errors
+      return true unless unfinished_participant_continue_training?
+
+      amend_cohort.save || save_error(amend_cohort.errors.full_messages.first)
```

Comment:
If this (line 9) fails for some reason have we lost the "error" so it won't be retried next time?